.github/workflows/setup-cf-env.yml

```diff
@@ -0,0 +1,95 @@
+name: Setup CF Environment
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+      capi-version:
+        required: true
+        type: string
+    outputs:
+      environment-name:
+        description: "Name of claimed environment"
+        value: ${{ jobs.setup-cf-env.outputs.environment-name }}
```

Comment:
this should be `${{ steps.claim-toolsmiths-env.outputs.environment-name }}` as we want the output of the step, not an out put of the job.  See [this](https://github.com/cloudfoundry/cli/pull/2326/files#r986032667) for original issue